app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  belongs_to :user, foreign_key: 'author_id'
+  belongs_to :post
+
+  validates :author_id, presence: true
+  validates :post_id, presence: true
+  validates :text, presence: true
+
+  def increment_post_comment_counter
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
- [ ] [OPTIONAL] What is the reason the format of the comment_counter is different than the format of the like and post counter? both work but it will make it a lot easier to read an update in the future if the code is the same if the functionality is the same.